README.md

```diff
@@ -1,34 +1,28 @@
-# The SpatialOS GDK for Unreal (pre-alpha)
+# The SpatialOS GDK for Unreal Engine (Alpha)
```

Comment:
I've had a little bit of a rejig to make the readme less long and wordy - also a few questions for you @girayimprobable.
Rather than try to add this as comments, here (which I think would be tricky)  I have added your amendments to mine in the GDoc here: https://docs.google.com/document/d/1tu_VEPJtfTl6F0r8QswuVvKtoDQyXovr9AOV4Uj3fkc/edit
I hope that this is OK.